refactor(curriculum): restructure sudoku solver python project (#54303)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/07-scientific-computing-with-python/learn-classes-and-objects-by-building-a-sudoku-solver/660a968ca0838773c9bbfc85.md

---
id: 660a968ca0838773c9bbfc85
title: Step 69
challengeType: 20
dashedName: step-69
---

# --description--

> When you print your `gameboard` object you get something like `<__main__.Board object at 0xf3c1c8>`, which is the default representation of an object. And this means that the `solve_sudoku` function will give you an output different from what you want, as well.

Reviewer: ```suggestion
When you print your `gameboard` object, you get something like `<__main__.Board object at 0xf3c1c8>`, which is the default representation of an object. This means that the `solve_sudoku` function will also give you an output different from what you expect.
```
